src/client/ns_turn_msg.c

```diff
@@ -263,7 +267,7 @@ int stun_produce_integrity_key_str(const uint8_t *uname, const uint8_t *realm, c
  		EVP_DigestUpdate(ctx,str,strl);
  		EVP_DigestFinal(ctx,key,&keylen);
  		EVP_MD_CTX_free(ctx);
-#else // OPENSSL_VERSION_NUMBER < 0x10100000L
+#else // OPENSSL_VERSION_NUMBER > 0x10100000L && OPENSSL_VERSION_NUMBER >= 0x30000000L
```

Comment:
Comment was and is incorrect. Maybe just remove it.
For protocol, it should say // OPENSSL_VERSION_NUMBER >= 0x10100000L && OPENSSL_VERSION_NUMBER < 0x30000000L